schemas/Manual.game.schema.json

```diff
@@ -80,9 +80,18 @@
                     "description": "(Optional) how many items of this block will be randomly added to inventory. Will add every item in the block if not included",
                     "type":"integer"
                 },
+                "yaml_option": {
+                    "description": "(Optional) Array of Options that will decide if the items & locations in this category are enabled",
```

Comment:
```suggestion
                    "description": "(Optional) Array of Options that will decide if this block is rolled",
```

Reply:
oops forgot to add the why for this sugested change:
in startinv if you look in the other defs you can see that its starting_items's blocks we define not items & locations